suite-common/message-system/src/messageSystemUtils.ts

```diff
@@ -302,10 +301,7 @@ export const getValidMessages = (config: MessageSystem | null, options: Options)
         .map(action => action.message);
 };
 
-export const getValidExperiments = (
-    config: MessageSystem | null,
-    options: Options,
-): ExperimentsItem[] => {
+export const getValidExperiments = (config: MessageSystem | null, options: Options): string[] => {
```

Comment:
Do I understand correctly that this functions returns experiment IDs? 🤔 

If yes, then I'd suggest different naming:
```suggestion
export const getValidExperimentIds = (config: MessageSystem | null, options: Options): string[] => {
```

It'd also be great to make the return type more strict if possible.

Reply:
456888977e4e680912979bc2d54e6a52632f4fcc Renamed. The config type is auto-generated so I would not use branded type here. It can be used, but I see much more important candidates for more strict types everywhere around.